### PR TITLE
Firebase - Case Study

### PR DESCRIPTION
**Pull request by May-Tove**

### CODE REVIEW

File: firebase-case-study.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+Firebase was founded in 2011 by James Tamplin and Andrew Lee and initially offered a real-time database for mobile and web applications. In 2012, Firebase launched its first product, a real-time backend for apps, which quickly became popular among developers. In 2014, Firebase was acquired by Google, which allowed the company to expand its offerings and integrate with other Google services. 
+
+Over the next several years, Firebase added new features, such as authentication, storage, hosting, and functions, becoming a complete platform for app development. In 2017, Firebase introduced Firebase ML Kit, a machine learning platform for mobile apps, further expanding its capabilities. Today, Firebase is one of the most popular cloud-based platforms for app development and is used by millions of developers worldwide.
+
+
+## Features
+
+As mentioned, there are many features in Firebase, which can be divided into three main products; build, release & monitor and engage. I will write about the features of each one in more detail below.
+
+<img width="806" alt="image" src="https://user-images.githubusercontent.com/89157761/218459770-11cf4e92-7bf0-4753-b7be-9869413e2855.png">
```
Comment (Fermain) [inline firebase-case-study.md:42]: Please change this to markdown image format: `alt text`

File: firebase-case-study.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+
+### Build - Get to market and deliver value to your users, faster
+
+**Cloud Functions**: Enables developers to automatically execute backend code in response to events brought on by Firebase features and HTTPS requests. They allow developers to run backend code without having to manage servers.
+
+**Authentication**: Aims to simplify the development of secure authentication systems while enhancing end users' sign-in and onboarding processes. It offers a complete identification solution, including login for Google, Twitter, Facebook, and GitHub in addition to email and password accounts.
```
Comment (Fermain) [inline firebase-case-study.md:49]: What is the underlying technology called?

**Review by Fermain — COMMENTED** [review]
This is a well researched and written case study that covers a large subject succinctly. Considering there are other firebase case studies in PR, it would be appreciated if you gave a review for one of these. 

I have prompted for more detail in a few instances.

**Comment by RobertDacian** [comment]
Hey, this is a nicely done case study about Firebase. I believe it's an excellent tool for a small project. You get up and running fast, but I would not recommend Firebase for a large project that needs to scale appropriately or have complete control over the backend. Overall it's still a way to go. Still not an all-in-one cookie cutter, but it's a great tool.

**Comment by May-Tove** [comment]
> Hey, this is a nicely done case study about Firebase. I believe it's an excellent tool for a small project. You get up and running fast, but I would not recommend Firebase for a large project that needs to scale appropriately or have complete control over the backend. Overall it's still a way to go. Still not an all-in-one cookie cutter, but it's a great tool.

Thank you for the feedback, Robert!

**Review by Fermain — APPROVED** [review]
A detailed and helpful case study, thanks!

**Pull request merged**